layouts/partials/scripts.html

```diff
@@ -1,11 +1,12 @@
+<script src="/js/jquery-3.6.0.min.js"  crossorigin="anonymous"></script>
+<script src="/js/popper-1.16.1.min.js" crossorigin="anonymous"></script>
+<script src="/js/bootstrap-4.6.1.min.js" crossorigin="anonymous"></script>
+<script src="{{ "js/script.js" | relURL }}"></script>
+<script async src="/js/mermaid.min.js" crossorigin="anonymous"></script>
 
-<script src="/js/popper-1.14.3.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
-<script src="/js/bootstrap-4.3.1.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
```

Comment:
Are we losing this `integrity` value from the replacement code? If so, why is that?

Reply:
We are. Seem to recall something said it was okay to omit. I'll fix stat. Tx.

Reply:
(Remember to mark this conversion as solved, thanks.)